lib/tinybucket/connection.rb

```diff
@@ -54,6 +49,20 @@ def default_middleware(_options)
       end
     end
 
+    def configure_auth(conn)
+      if Tinybucket.config.access_token
+        conn.request :oauth2, Tinybucket.config.access_token
+      else
+        oauth_secrets = {
+          consumer_key:    Tinybucket.config.oauth_token,
+          consumer_secret: Tinybucket.config.oauth_secret
+        }
+
+        conn.request :oauth, oauth_secrets
+        conn.response :follow_oauth_redirects, oauth_secrets
```

Comment:
I created the `follow_oauth_redirects` middleware to support OAuth1.0 signed request on HTTP redirection.
So this middleware does not need in OAuth2.0 😄